tests/docs/running-perf-tests.md

```diff
@@ -140,6 +141,21 @@ make setup-test-env-k6
 make test-perf-all
 ```
 
+You can also run the test apps individually.
+```bash
+# build perf apps docker image under apps/
+make test-perf-<app-name>
+
+# push perf apps docker image to docker hub
+make test-perf-<app-name>
+```
+`<app-name>` can be found at [PERF_TESTS](https://github.com/dapr/dapr/blob/6def7d1b9ffe896b7b06d05128b9cd605d39f939/tests/dapr_tests.mk#L70)
+
+For example, if you only want to run workflows tests, you can run
+```bash
+make test-perf-workflows
+```
+
```

Comment:
Also a helpful command is "make test-clean" which removes all test data so it's easier to find the new one!

Reply:
updated as suggested.